benchmarks/runners/xspace_bench_bench.py

```diff
@@ -31,7 +31,7 @@ def benchmark_nlo(self):
             ["ToyLHAPDF"],
         )
 
-
+@pytest.mark.skip
```

Comment:
`pytest` should always run all the provided benchmarks, previous occurences of `skip` where just temporary, because we did not have the time of making them work properly. Since you are exactly making them work, remove any `skip` before merging and test everything you wrote.